main.cc

```diff
@@ -913,6 +913,7 @@ run()
             glClearColor(0, 0.18f, 0.21f, 1);
             glClear(GL_COLOR_BUFFER_BIT);
             ImGui::Render();
+            ImGui_ImplSdlGL3_RenderDrawData(ImGui::GetDrawData());
```

Comment:
This isn't wonderful to have to do :(

Reply:
Came with the new stuff. I'm not sure if the callback thing is supported anymore. That said, we could probably wrap in a ImGui_ImpSdlGL3_Render()?